pool/type.gno

```diff
@@ -171,7 +172,7 @@ func (t *TickInfo) init() {
 
 type Ticks map[int32]TickInfo          // tick => TickInfo
 type TickBitmaps map[int16]*u256.Uint  // tick(wordPos) => bitmap(tickWord ^ mask)
-type Positions map[string]PositionInfo // positionKey => PositionInfo
+type Positions map[string]PositionInfo // positionKey => PositionInfo 
```

Comment:
Aren't we supposed to replace all map to avl?
Since you imported avl from #L8 but doesn't using it. Maybe its WIP??

Reply:
All the PRs are tangled together, I think avl must be apply last.